src/arcade/src/Microsoft.DotNet.Build.Tasks.Installers/build/wix/wix.targets

```diff
@@ -452,24 +452,10 @@
         ComponentMsiFile=$(CrossArchMsiFile)" />
   </Target>
 
-  <!--
-    Acquire NuGet.exe, if not present.
-  -->
-  <Target Name="_AcquireNuGetExe">
-    <MSBuild
-      Projects="$(MSBuildThisFileDirectory)..\acquisition\acquire-nuget-exe\acquire-nuget-exe.proj"
```

Comment:
Can you also remove `acquire-nuget-exe.proj`? There should be no other usages of that file any more. If there are, please migrate the rest of the usages to use `dotnet pack <nuspec>`.